trex-txrx.py

```diff
@@ -20,8 +20,131 @@ def ip_to_int (ip):
     ip_int = 256**3 * int(ip_fields[0]) + 256**2 * int(ip_fields[1]) + 256**1 * int(ip_fields[2]) + int(ip_fields[3])
     return ip_int
 
+def calculate_latency_pps (dividend, divisor, total_rate):
+     return int((float(dividend) / float(divisor) * total_rate))
+
+def create_traffic_profile (direction, measure_latency, default_stream_pg_id_base, latency_stream_pg_id_base, latency_rate, frame_size, num_flows, src_mac_flows, dst_mac_flows, src_ip_flows, dst_ip_flows, mac_src, mac_dst, ip_src, ip_dst):
+     streams = { 'default': { 'pg_ids': [], 'names': [], 'frame_sizes': [], 'traffic_shares': [] }, 'latency': { 'pg_ids': [], 'names': [], 'frame_sizes': [], 'traffic_shares': [] } }
+
+     if frame_size == "imix":
+          # imix is defined as 7 packets of size 40 bytes, 4 packets of size 576 bytes, and 1 packet of size 1500 bytes
+          # from https://en.wikipedia.org/wiki/Internet_Mix
+
+          small_packets = 7
+          medium_packets = 4
+          large_packets = 1
+          total_packets = small_packets + medium_packets + large_packets
+
+          small_packet_bytes = 40
```

Comment:
Good catch.  I completely misread the page and interpreted "incl. IP header" as that was the total frame size.  I'll fix it.